langkit/input_output.py

```diff
@@ -53,3 +37,25 @@ def prompt_response_similarity(text):
             )
             series_result.append(None)
     return series_result
+
+
+def init(
+    language: str = "en"
+    transformer_name: Optional[str] = None,
+    custom_encoder: Optional[Callable] = None,
+    config: Optional[LangKitConfig] = None,
+):
+    config = config or deepcopy(lang_config)
+    global _transformer_model
+    if transformer_name is None and custom_encoder is None:
+        transformer_name = config.transformer_name
+    _transformer_model = Encoder(transformer_name, custom_encoder)
+    register_dataset_udf(
+        [_prompt, _response],
+        f"{language}.{_response}.relevance_to_{_prompt}",
```

Comment:
this renaming prefixing the language in the metric name will create a discontinuity with existing integrations and break back-compat.

We shouldn't prefix the localization in the metric name, at least not for the original english only launch of LangKit. Better would be to put this in metadata or in the platform something like column entity schema?

Reply:
Do you want, for example, to track English and French toxicity in the same column?

Reply:
maybe we could keep the original name for english, and add the language prefix only for other languages?